packages/destination-actions/src/destinations/braze/updateUserProfile/__tests__/__snapshots__/snapshot.test.ts.snap

```diff
@@ -13,7 +13,7 @@ Object {
       },
       "date_of_first_session": "2021-02-01T00:00:00.000Z",
       "date_of_last_session": "2021-02-01T00:00:00.000Z",
-      "dob": "2021-02-01",
+      "dob": "2021-01-31",
```

Comment:
do we need this? Might need to rebase your local branch off of `main`: https://git-scm.com/docs/git-rebase